Added `databricks_model_serving` support to `databricks_permissions` (#2331)
databricks/terraform-provider-databricks: docs/resources/model_serving.md

* `served_model_name` - (Required) The name of the served model this route configures traffic for. This needs to match the name of a `served_models` block
* `traffic_percentage` - (Required) The percentage of endpoint traffic to send to this route. It must be an integer between 0 and 100 inclusive.

## Attribute Reference

In addition to all arguments above, the following attributes are exported:

> * `id` - Serving endpoint identifier (equal to `name` argument).

Reviewer: ```suggestion
* `id` - Equal to the `name` argument and used to identify the serving endpoint.
```
Just making it more clear that this attribute is effectively a dupe and to make it easier to distinguish from `serving_endpoint_id`.

> * `serving_endpoint_id` - unique identifier of the serving endpoint that could be used to set permissions and other operations.

Reviewer: ```suggestion
* `serving_endpoint_id` - Unique identifier of the serving endpoint primarily used to set permissions and refer to this instance for other operations.
```
